all-repos.yaml

```diff
@@ -1,173 +1,174 @@
 # This file is used to generate all-hooks.json
-- https://github.com/pre-commit/pre-commit-hooks
-- https://github.com/pre-commit/mirrors-autopep8
-- https://github.com/pre-commit/mirrors-coffeelint
-- https://github.com/pre-commit/mirrors-csslint
-- https://github.com/pre-commit/mirrors-eslint
-- https://github.com/pre-commit/mirrors-fixmyjs
-- https://github.com/pre-commit/mirrors-jshint
-- https://github.com/pre-commit/mirrors-mypy
-- https://github.com/pre-commit/mirrors-prettier
-- https://github.com/pre-commit/mirrors-puppet-lint
-- https://github.com/pre-commit/mirrors-ruby-lint
-- https://github.com/pre-commit/mirrors-scss-lint
-- https://github.com/pre-commit/mirrors-yapf
-- https://github.com/pre-commit/pygrep-hooks
-- https://github.com/FalconSocial/pre-commit-mirrors-pep257
-- https://github.com/FalconSocial/pre-commit-python-sorter
-- https://github.com/guykisel/pre-commit-robotframework-tidy
-- https://github.com/guykisel/prospector-mirror
+# - https://github.com/hakancelik96/unimport
```

Comment:
this file is intentionally not sorted, please append your repository to the bottom

Reply:
👍  will append to end. Out of curiosity: why not sorted? (could help in duplicate detection?)

Reply:
it populates https://pre-commit.com/hooks.html

Reply:
hmm then won't the page be alphabetically populated?

Reply:
pre-commit doesn't start with an "a" does it?